enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/discovery/HazelcastCoreTopologyService.java

```diff
@@ -140,6 +142,16 @@ public void setLeader( LeaderInfo leaderInfo, String dbName )
         }
     }
 
+    @Override
+    public void handleStepDown( LeaderInfo leaderInfo, String dbName )
```

Comment:
Maybe `newLeaderInfo` would be a better name, to distinguish it from the `leaderInfo` field?